beacon_node/beacon_chain/src/attestation_verification.rs

```diff
@@ -547,8 +560,28 @@ impl<'a, T: BeaconChainTypes> IndexedAggregatedAttestation<'a, T> {
                 let selection_proof =
                     SelectionProof::from(signed_aggregate.message.selection_proof.clone());
 
+                let factor = if chain
+                    .spec
+                    .fork_name_at_slot::<T::EthSpec>(signed_aggregate.message.aggregate.data.slot)
+                    >= ForkName::Deneb
+                {
+                    let total_committee_balance = chain
+                        .effective_balances_cache
+                        .get_committee_balance::<T::EthSpec>(&committee)
+                        .map_err(|e| Error::BeaconChainError(e))?;
+                    let aggregator_balance = chain
+                        .effective_balances_cache
+                        .get_effective_balance(relevant_epoch, aggregator)
+                        .map_err(|e| Error::BeaconChainError(e))?;
+
+                    // TODO: is it just standard integer division?
+                    (total_committee_balance / aggregator_balance) as usize
```

Comment:
Yes standard division, this should be safe div as aggregator_balance could be 0 is some bizarre cases